src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -107,7 +107,7 @@ const ShippingRouteLayout: FC<
             }
           >
             {expressCheckoutPrototypeEnabled &&
-              !shippingContext.orderData.isOffer && (
```

Comment:
Personally I'd put this all in one variable in this file: feature flag plus flat fee shipping plus no offers. It's not a problem going in context, but we don't need it anywhere except this one spot.